kikar_hamedina/facebook_feeds/models.py

```diff
@@ -441,6 +441,17 @@ def is_youtube_video(self):
         return False
 
     @property
+    def video_source_type(self):
```

Comment:
This is a mistake. it is not relevant. pls do not update the main repository with this file.